lib/stream.js

```diff
@@ -80,6 +80,19 @@ module.exports = class UDXStream extends streamx.Duplex {
     return this._view[binding.offsetof_udx_stream_t_cwnd >> 2]
   }
 
+  get retransmitCount () {
+    return this._view[binding.offsetof_udx_stream_t_retransmit_count >> 2] & 0xffff
+  }
+
+  get fastRecoveryCount () {
+    return this._view[binding.offsetof_udx_stream_t_fast_recovery_count >> 2] & 0xffff
+  }
+
+  get rtoCount () {
+    // rto = Retransmission timeout
```

Comment:
Please confirm that this is indeed the retransmission timeout (I had to look it up, so not 100% sure)

Reply:
dont think this should be exposed, the rto is simply based on the rtt which is already exposed